cmd/utils.go

```diff
@@ -619,6 +620,8 @@ func getConfigLabels(projectConfig ProjectConfig) (map[string]string, error) {
 	if projectConfig.License != "" {
 		if valid, err := IsValidKubernetesLabelValue(projectConfig.License); !valid {
 			return labels, errors.Errorf("%s license value is invalid. %v", ConfigFile, err)
+		} else if err := checkValidLicense(projectConfig.License); err != nil {
+			return labels, errors.Errorf("The .appsody-config.yaml SPDX license ID is invalid: %v.", err)
```

Comment:
I'm pretty sure this can also be the `stack.yaml` - when called from stack package.

Reply:
Sorry @skoh7645 I'm not being clear. This is a utility method that can be called with information from either `.appsody-config.yaml` or `stack.yaml`. Therefore you cannot hardcode the filename here, you'll have to figure out a different way to print it in the message.

Reply:
Sorry @kylegc for not understanding! I didn't realise we allowed users to add to the `.appsody-config.yaml` file to add their own labels. I have addressed this in the most recent commit